crawl-ref/source/viewgeom.h

```diff
@@ -22,6 +22,7 @@ class crawl_view_buffer
 public:
     crawl_view_buffer();
     crawl_view_buffer(const coord_def &sz);
+    crawl_view_buffer(crawl_view_buffer const&) = default;
```

Comment:
I don't like the placement of `const` here. The line above has it the other way round. Be consistent.